### DISCUSSION TITLE
Cannot install all extensions?

### DISCUSSION THREAD

**Discussion by trymeouteh** [VSCodium/vscodium#1129] [Q&A]
I tried to install this [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) Extension in VSCodium but when I search for "ESLint" I do not get the same extension which I am able to tell by the author of the extension and how many downloads the extension has.

When I search "ESLint" in Visual Studio Code, I do get the ESLint extension I am looking for. Why is this the case? Do I need to enable a setting? Is the now compatible with VSCodium?

**Comment by trymeouteh** [comment]
Extensions need to be published on Open VSX by submitting an request

https://github.com/open-vsx/publish-extensions/issues

**Comment by vinyanalista** [comment]
https://github.com/EclipseFdn/publish-extensions/issues/786
